dev/io.openliberty.com.sun.xml.messaging.saaj/bnd.overrides

```diff
@@ -0,0 +1,38 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.com.sun.xml.messaging.saaj
+
+Include-Resource: \
+  @${repo;com.sun.xml.messaging.saaj:saaj-impl;2.0.0}!/META-INF/services/jakarta.xml.soap.SOAPFactory, \
+  @${repo;com.sun.xml.messaging.saaj:saaj-impl;2.0.0}!/META-INF/services/jakarta.xml.soap.SOAPConnectionFactory, \
+  @${repo;com.sun.xml.messaging.saaj:saaj-impl;2.0.0}!/META-INF/services/jakarta.xml.soap.SAAJMetaFactory, \
+  @${repo;com.sun.xml.messaging.saaj:saaj-impl;2.0.0}!/META-INF/services/jakarta.xml.soap.MessageFactory
+ 
+ 
+Export-Package: com.sun.xml.messaging.saaj;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.packaging.mime;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.packaging.mime.internet;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.packaging.mime.util;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.soap;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.soap.dynamic;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.soap.name;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.util;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.util.stax;version="2.0.0";thread-context=true,\
+  com.sun.xml.messaging.saaj.util.transform;version="2.0.0";thread-context=true,\
```

Comment:
You can remove the end of line characters here `,\` since there's nothing after the last line right?

Reply:
done!